docs/source/blocks/core/grid.md

```diff
@@ -0,0 +1,169 @@
+---
+myst:
+  html_meta:
+    "description": "Volto grid block description and developer notes"
+    "property=og:description": "Volto grid block description and developer notes"
+    "property=og:title": "Volto grid block"
+    "keywords": "Volto, React, blocks, grid, container, Plone"
+---
+
+(grid-block)=
+
+# Grid block
+
+```{versionadded} Volto 17.0.0-alpha.10
+```
+
+Volto comes with a grid block.
+It builds upon the {doc}`container`.
+It is a block that contains other blocks.
+The grid block uses a fixed horizontal layout with a configurable maximum number of blocks, with a default of four blocks.
+
+The grid block can contain any block type.
+However, due to its horizontal alignment and narrow widths of the blocks it contains, the blocks might need additional styling.
+For this reason, only four blocks are included by default, namely `slate` (text), `image`, `listing`, and `teaser`.
+
+
+## Settings
+
+The grid block has the following settings.
+
+`Headline`
+:   The headline is a heading that appears above the title of the grid block.
```

Comment:
@stevepiercy It's not "above the title". It is the title. We already fixed this before your latest edits. (Thanks for those, btw.)
```suggestion
:   The headline is a heading that appears above the columns of the grid block.
```